zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/common/ElementTreePathBuilderTest.java

```diff
@@ -100,16 +113,26 @@ public void shouldIncludeProcessDefinitionAndCallingElementTreePaths() {
     final ElementInstance processB =
         elementInstanceState.newInstance(
             102, processBRecord, ProcessInstanceIntent.ELEMENT_ACTIVATING);
+    processState.putProcess(
+        processBRecord.getProcessDefinitionKey(),
+        createChildProcess(
+            processBRecord.getProcessDefinitionKey(), "B", ServiceTaskBuilder::done));
```

Comment:
💭 This resolves a problem that appears from the main change in this commit. Personally, I'd extract this into another commit to clarify intentions better, both to the reviewer as well as future readers.

Reply:
I described the intention of this change in the last line of the commit description:
> Also updated ElementTreePathBuilderTes to also populate processState since now it's needed to build callingElementPath

I could have separated the change in tests from the change in the builder, but I think they are relevant together, since the need for the processState was in the tree path builder was introduced in this commit.